src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2487,5 +2471,15 @@ public virtual void ApplyStyle(ViewStyle viewStyle)
                 }
             }
         }
+
+        /// <summary>
+        /// Used to restore the transition.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
In TCSACR-395, I couldn't see any "Add-ACR". so I understand that this is added as Hidden-API, then please **REMOVE** this comment. : `/// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.`
thank you.